### PR TITLE
Add support for Fuji's CropMode tag (0.27->master)

### PR DESCRIPTION
**Pull request by FreddieWitherden**
This small PR adds support for a missing tag in the Fuji Makernotes that I've discovered I need for my lens correction work.  Details for the tag were taken from https://exiftool.org/TagNames/FujiFilm.html

### CODE REVIEW

File: src/fujimn_int.cpp [hunk 1]
```diff
@@ -192,6 +192,14 @@ namespace Exiv2 {
         { 32768, N_("Film simulation mode") }
     };
 
+    //! CropMode, tag 0x104d
+    extern const TagDetails fujiCropMode[] = {
+        { 0, N_("None")                          },
+        { 1, N_("None")                          },
```
Comment (kmilos) [inline src/fujimn_int.cpp:198]: These don't match the translated values ExifTool thinks are right... Do you have more references?

**Review by kmilos — COMMENTED** [review]
Translated values are incomplete?

File: src/fujimn_int.cpp (hunk 1 above)
Comment (clanmills) [reply src/fujimn_int.cpp:198]: screenshot_25

Comment (clanmills) [reply src/fujimn_int.cpp:198]: screenshot_25

**Comment by FreddieWitherden** [comment]
> Translated values are incomplete?

Good spot.  I have fixed the issue when the tag value is one.

As an aside, when this tag is 2 or 4 the image has an additional 1.25 crop factor attached to it.  However, this does not appear to be reflected in the `Exif.Photo.FocalLengthIn35mmFilm` tag written out by the camera (it assumes the crop is 1.5 rather than 1.5*1.25).  I am unsure how we want to handle this.

**Comment by kmilos** [comment]
> this does not appear to be reflected in the `Exif.Photo.FocalLengthIn35mmFilm` tag written out by the camera

It's a job for Exiv2 to "fix" this IMHO.

**Comment by clanmills** [comment]
Let's try to minimise "interpreting" the metadata.  It's like the lens recognition which should never have been in Exiv2.  It has created end-less work.  So, let's do as little as possible.

If you have to do this "when there is an R in the month, multiply by 1.25, you'll have to do it in code by supplying a specialised `PrintFct`.  When we print the value (not the interpreted value), we print the value in the metadata.

**Comment by clanmills** [comment]
There is a PrintFnc already defined: `print0xa405`.  We shouldn't touch this.  

```bash
862 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ exiv2 -pt -g 35  ~/Downloads/sample1.raf 
Exif.Photo.FocalLengthIn35mmFilm             Short       1  83.0 mm
863 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ 
```

The tag is written by the manufacturer in Exif.Photo.FocalLengthIn35mmFilm.  All the PrintFct does is to decorate the output:

```cpp
    std::ostream& print0xa405(std::ostream& os, const Value& value, const ExifData*)
    {
        long length = value.toLong();
        if (length == 0) {
            os << _("Unknown");
        }
        else {
            os << length << ".0 mm";
        }
        return os;
    }
```

**Review by kmilos — APPROVED** [review]

**Comment by clanmills** [comment]
Thanks you, Gentlemen for working on this.

**Pull request merged**